src/access_py_telemetry/api.py

```diff
@@ -150,9 +171,9 @@ def send_api_request(
                 f"Endpoint for '{service_name}' not found in {self.endpoints}"
             ) from e
 
-        endpoint = f"{self.server_url}{endpoint}"
+        endpoint = str(PurePosixPath(self.server_url) / endpoint.lstrip("/"))
```

Comment:
This seems to change `http://` to one-slash `http:/`

```
>>> PurePosixPath("https://tracking-services")
PurePosixPath('https:/tracking-services')
```

Reply:
```suggestion
        endpoint = f"{self.server_url.rstrip('/')}/{endpoint.lstrip('/')}"
```
A f-string might work here?

Reply:
Good spot, I didn't notice that.

I think strip will match multiple slashes - eg. `'abc///'.rstrip('/') == 'abc'`. Not sure this will ever be the case or matter in practice but I've just used `.replace` in case it does.

Reply:
Ooh good point, replace is probably safer then